guides/multiSigtx-creation-signing/create-multiSig-transaction.js

```diff
@@ -0,0 +1,113 @@
+const { apiClient, cryptography, codec, Transaction, transactions } = require('lisk-sdk');
+const { transferParamsSchema } = require('lisk-framework/dist-node/modules/token/schemas');
+const { accounts } = require('./accounts.json');
+const signedTX = require('./signedTx.json');
+const readline = require("readline");
+const fs = require("fs");
+
+const RPC_ENDPOINT = 'ws://127.0.0.1:7887/rpc-ws';
+let privateKeyStr;
+let existingSignedTx;
+const chainID = Buffer.from('00000001', 'hex');
+const optionalAccount1 = accounts[0];
+const optionalAccount2 = accounts[1];
+const optionalAccount3 = accounts[2];
+const sortedOptionalKeys = [Buffer.from(optionalAccount1.publicKey, 'hex'), Buffer.from(optionalAccount2.publicKey, 'hex'), Buffer.from(optionalAccount3.publicKey, 'hex')].sort((a, b) => a.compare(b));
+const sortedMandatoryKeys = [];
+const senderKeyInfo = accounts[0];
+const latestNonce = BigInt(3);
+
+const keys = {
+    mandatoryKeys: sortedMandatoryKeys,
+    optionalKeys: sortedOptionalKeys,
+};
+
+const tokenTransferParams = {
+    tokenID: Buffer.from('0000000100000000', 'hex'),
+    recipientAddress: cryptography.address.getAddressFromLisk32Address(accounts[1].address),
+    amount: BigInt(20000000),
+    data: 'Hello!',
+};
+
+const encodedTransferParams = codec.encode(transferParamsSchema, tokenTransferParams);
+
+const unSignedTx = new Transaction({
+    module: 'token',
+    command: 'transfer',
+    nonce: latestNonce,
+    senderPublicKey: Buffer.from(senderKeyInfo.publicKey, 'hex'),
+    fee: BigInt(1000000),
+    params: encodedTransferParams,
+    signatures: [],
+});
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+if (process.argv.length < 3) {
+    console.log("Please provide all the required parameter when executing the script:");
+    console.log("node create-multiSig-transaction-offline.js PRIVATEKEY");
+    process.exit(1);
+}
+
+fs.readFile('signedTx.json', (err, data) => {
+    if (err) {
+        existingSignedTx = '';
+    }
+    else {
+        existingSignedTx = data.byteLength;
+    }
+})
+
+rl.question("Do you want to sign the transaction? 'yes'", function (confirmed) {
+    confirmed = confirmed.toLowerCase();
+    if (confirmed == "yes" || confirmed == "y") {
+        process.argv.forEach(function (val, index) {
+            if (index === 2) {
+                privateKeyStr = val;
+            }
```

Comment:
maybe no loop needed here?
```suggestion
            privateKeyStr = process.argv[2];
```